common/changes/@microsoft/rush/chao-fix-edge-case_2024-05-18-05-52.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Refactor the alwaysInjectDependenciesFromOtherSubspaces logic.",
```

Comment:
Is this refactoring relevant to a Rush user? If it is, can you update this to explain what the changes are?